PSSwagger/SwaggerUtils.psm1

```diff
@@ -1471,18 +1471,14 @@ function Get-PathFunctionBody
         [string]
         $HostOverrideCommand,
```

Comment:
This is not used anywhere in this file.  Should it be removed?

Reply:
As we discussed offline, this variable will be used for non-Azure services in below line.
```powershell
$body = $executionContext.InvokeCommand.ExpandString($functionBodyStr)
```